intro-Cloud/ch_LaunchInstances.tex

```diff
@@ -92,6 +92,10 @@ \section{Launch an instance}\label{launch-an-instance}
 
 \item[Networks] tab. Add one or more networks to the instance.
 
+  \strong{Note:} Since the user's project dedicated IP (or the selected floting IP)
+  is associated with the _vsc network, the latter must be selected should the user
```

Comment:
[quote]
you should always add _vm network to get access to internet from your VM (also if you want to get access from external networks via project's public floating IP). Projects with shared file systems or VSC network should add _nfs and _vsc networks here as well (besides _vm) in order to connect to these services (see section 5.1 for more information)
[/quote]

How does one connect to a VM with only _vm network from external networks, e.g., with VSC? This is not clear, also to me. In fact, the suggested edit (adding a _vsc network) was discussed with a user who did not have a project, but wanted to ssh to his VM from the UGent login node. And that was exactly what the solution. So that was me not understanding how things work. Point taken.

Than could we simply add/change to:

\strong{Note:} The _vsc network must be selected should the user wish to access the instance from within the VSC.

Reply:
@avapirev the usual way to connect to a VM is using the public IP (with *_vm network) via floating ip (with our without port forwarding) but that is not explained in this section, it has its specific section https://hpcugent.github.io/vsc_user_docs/pdf/intro-Cloud.pdf#section.5.2 but maybe here we can inform the users they must choose _vm and then for external access they should go to section 5.2. About VSC network this is only available if the project requests this to deploy services within vsc network, and indeed its easier to connect directly using vsc floating ips but this is not generic, most of cloud projects do not have access to vsc net, we should clarify that.

Reply:
Alright, I found which part of the manual must have caused the initial confusion. In 5.3 Security Groups there it says:

[quote]
In the VSC cloud, each project contains a default security group, which allows you to ping instances and connect using SSH on the default port 22.
[/quote]

I don't see how that's possible without adding "ssh"/"ssh access" to the instance. At least it's not very clear. Otherwise, going more carefully over the manual in its current form, I managed to create and instance with a "_vm" network and "ssh"/"ssh access" security groups and connected via ssh.

Anything I am missing? Otherwise I am happy with the rest of the documentation.